libraries/common/include/common/util.hpp

```diff
@@ -73,6 +73,7 @@ bool write(stream &stm, T const &value) {
 void thisThreadSleepForSeconds(unsigned sec);
 void thisThreadSleepForMilliSeconds(unsigned millisec);
 void thisThreadSleepForMicroSeconds(unsigned microsec);
+dev::bytes makeVrfInput(taraxa::level_t level, const dev::h256 &period_hash);
```

Comment:
I think this should be static function dag_block class as it is related to dag only

Reply:
or in vdf somewhere